crates/gossip/src/gossip.rs

```diff
@@ -717,42 +652,6 @@ mod test {
             .await
             .unwrap();
 
-        harness_2
-            .gossip
-            .dht
-            .write()
-            .await
-            .inform_ops_stored(vec![op_2.clone().into()])
-            .await
-            .unwrap();
-
-        // Inform the harnesses that we've already synced up to now. This will force the
-        // ops we just created to be treated as historical disc ops.
-        harness_1
-            .gossip
-            .peer_meta_store
-            .set_new_ops_bookmark(
```

Comment:
Oh this is why the block above isn't needed. This test has just been reduced to a "what's new" check